patches/kex_rlwe_newhope_avx2/patch0.patch

```diff
@@ -0,0 +1,18 @@
+diff --git a/src/kex_rlwe_newhope/avx2/Makefile.am b/src/kex_rlwe_newhope/avx2/Makefile.am
+new file mode 100644
+index 0000000..baced33
+--- /dev/null
++++ b/src/kex_rlwe_newhope/avx2/Makefile.am
```

Comment:
Same here, should that be in the patch?

Reply:
Yes, because autoconf requires the presence of Makefile.am whether or not the algorithm is enabled or not.